docs/product-getting-started-guide/src/main/asciidoc/test-suspend-proc.adoc

```diff
@@ -0,0 +1,46 @@
+[id='testing_suspend-proc']
+= Testing the driver license suspension scenario
+
+The following chapter instructs you how to test the driver license suspension scenario using the data that you specified when you set the Driver License Suspension rules and actions.
+
+[float]
+== Prerequisites
+
+* Created the Driver department traffic violations project
+* Created the Violation and Driver data models
```

Comment:
data objects instead of data models
And for all three bullets, monospace (` `) for project name and data object names

Reply:
good catch. done